### DISCUSSION TITLE
Cannot output logs into MySql

### DISCUSSION THREAD

**Discussion by nicola907** [fluent/fluent-bit#8477] [General]
Hi everyone, I want to use the "td-agent-bit" service to output logs into MySql. I have installed MySql and it runs on port 3306 (I left the default port). Then I created a database named "logs_db". Inside the "logs_db" database I created a table named "logs_tb" with a column "text" of type varachar(255). I configured the output section of my "td-agent-bit.conf" file like this.

```
[OUTPUT]
    Name        mysql
    Match       *
    Host        localhost
    Port        3306
    Database    logs_db
    Table       logs_tb
    Column      text
    User        root
    Password    d3qfeftgwff
```
Then I restarted the "td-agent-bit" service but it gives an error:

```
Feb 12 08:23:57 fluent-bit-d-machine systemd[1]: td-agent-bit.service: Start request repeated too quickly.
Feb 12 08:23:57 fluent-bit-d-machine systemd[1]: td-agent-bit.service: Failed with result 'exit-code'.
Feb 12 08:23:57 fluent-bit-d-machine systemd[1]: Failed to start TD Agent Bit.
```
The "mysql" service run properly. So what is wrong?

**Comment by nicola907** [comment]
I fear that MySql is not supported by Fluent Bit. In fact when I launched the command "fluent-bit -H" to see all the available plugins I didn't see mysql, but I saw pgsql. Therefore I tried PostgreSql and it worked. The correct configuration is the following

    [OUTPUT]
        Name                pgsql
        Match               *
        Host                localhost
        Port                5432
        User                postgres
        Password            j382h4dn223834oj4dn490
        Database            logs_db
        Table               logs_tb

"postgres" is the default user. "logs_db" is a database that must be created manually. Meanwhile the table "logs_tb" is created automatically. In the configuration we only have to indicate the name that it will be given, but we are not supposed to create it ourselves.

**Reply by patrick-stephens** [reply]
Yup, there's no mysql output: https://docs.fluentbit.io/manual/pipeline/outputs
Your logs will probably show you an error along the lines of `you've asked for an output that does not exist` hence the errors - you can check with journald to see the full log rather than the last status output from systemd.

On another note, `td-agent-bit` has been deprecated and removed for quite a while now so it also sounds like you have an old package: https://docs.fluentbit.io/manual/installation/linux#migration-to-fluent-bit
